config/routes.rb

```diff
@@ -1,7 +1,9 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users , path: 'show'
```

Comment:
Ok. makes sense.  I had the path in there before I had root designated `user#show` as the root, at the time it's the only way I thought I could access `show`, was to create a path to it.